lib/jekyll-relative-links/generator.rb

```diff
@@ -7,7 +7,7 @@ class Generator < Jekyll::Generator
     # Use Jekyll's native relative_url filter
     include Jekyll::Filters::URLFilters
 
-    LINK_TEXT_REGEX = %r!([^\]]+)!
+    LINK_TEXT_REGEX = %r!([^\]]+|.*)!
```

Comment:
If I'm reading this correctly, this is "any character other than `]` OR any character". Is there an advantage to that over just `.*`  or `.*?`? It looks like this still allows us to support multiple links on the same line?